### DISCUSSION TITLE
I would like to know if the module supports zhCN clients

### DISCUSSION THREAD

**Discussion by Mrming1994** [benjymansy123/mod-worgoblin#45] [General]
I tried packaging it into project code but it didn't work

**Comment by heyitsbench** [comment]
I've been told that zhCN/zhTW clients have issues when loading custom patches, but I'm not actually certain if that is true. If it is true, then it's unlikely to work full stop, but if that isn't true then I *believe* it should work fine for the most part. I think only a few files are left to finish localization with (can keep up with #31 for that), but the majority are localized (hopefully correctly) for each client, so it should work with any and all locales, again, with only a few exceptions.

**Reply by Mrming1994** [reply]
I did have problems loading custom patches on the zhCN/zhTW client 
The game will report an error when running client

**Reply by Mrming1994** [reply]
I really hope to play the module you made on zhCN/zhTW client, please

**Reply by heyitsbench** [reply]
I unfortunately don't have a way to patch the zhCN client to be able to use custom patches (especially since I don't have a zhCN client at all) but I can vouch for downloading an enUS client and copying the zhCN folder in the Data folder to that client, and swapping the language either in-game or through the Config.WTF file, as that's what has worked for me in the past.
